packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -336,4 +336,37 @@ export class ProjectActions {
       content: newSpec.content,
     }
   }
+
+  async scaffoldIntegration () {
```

Comment:
I think with the addition of this we could probably break this and the other CodeGen related methods out to another Action file that better encomppasses what they were doing. Maybe something like CodeGenActions

Reply:
I agree, and I think it could be further improved by moving a lot of stuff out of ProjectDataSource that is related to codegen. I'll tackle that after the create spec flow is in though.